spriint10/tema3.js

```diff
@@ -0,0 +1,16 @@
+// am definit o functie numita countLetter care primeste cele doua argumente
+function countLetter(str, letter) {
+// aici am initializat variabila count la 0 pentru a tine evidenaa numarului de aparitii
+let count = 0;
+// aici am folosit acest for pentru parcurgerea fiecarui caracter din sir
+        for (let i = 0; i < str.length; i++) { 
+// se va verifica cu ajutorului acestui if dacă caracterul str[i] este egal cu litera căutată
+          if (str[i] === letter) {
```

Comment:
In cazul in care o litera este mare iar alta mica, egalitatea aceasta nu se va intampla. E nevoie sa le transformi pe ambele in litera mica sau mare